util/icat_db_generator.py

```diff
@@ -507,16 +526,18 @@ class InvestigationParameterGenerator(Generator):
     amount = InvestigationGenerator.amount
 
     def generate(self):
-        self.pool_map(generate_investigation_parameter)
+        for i in range(1, self.amount):
+            generate_investigation_parameter(i)
 
 
 def generate_investigation_parameter(i):
+    # TODO - Why's this not in the class?
```

Comment:
Beats me. I added it back in as a static method and didn't see any difference in generated data. Wanna add it back?

Reply:
I forgot to search TODO in VS Code! The inconsistency irked me slightly (it makes no actual difference, but why's the function put in a different place to all the others), I shall resolve now

Reply:
Change made and TODO removed :)